src/Agent/NewRelic/Agent/Core/Config/ConfigurationLoader.cs

```diff
@@ -434,6 +434,12 @@ public static void PublishDeserializedEvent(configuration config)
             EventBus<ConfigurationDeserializedEvent>.Publish(new ConfigurationDeserializedEvent(config));
         }
 
+        // primarily for testing in CompositeAgent
```

Comment:
Can we make this a `///` style comment so that intellisense can help us to understand that this shouldn't be used in production code, and it is only present for the tests?